lib/internal/crypto/random.js

```diff
@@ -119,6 +120,74 @@ function randomFill(buf, offset, size, cb) {
   _randomBytes(buf, offset, size, wrap);
 }
 
+// Largest integer we can read from a buffer.
+// e.g.: Buffer.from("ff".repeat(6), "hex").readUIntBE(0, 6);
+const RAND_MAX = 281474976710655;
+
+function randomInt(min, max, cb) {
```

Comment:
I don't think we should be adding new callback apis. I'm thinking `randomInt() -> Promise<n>` and `randomIntSync() -> n`

Reply:
Tbh, I’d personally prefer consistency with `randomBytes()`. We can totally add `crypto.promises` like we did with fs/dns.

Reply:
@addaleax:
> We can totally add `crypto.promises`....

Heh... if I can get around to finishing that up I'm about 85% of the way ;-)